pkg/client/builds.go

```diff
@@ -65,10 +66,10 @@ func (c *builds) Create(build *buildapi.Build) (result *buildapi.Build, err erro
 }
 
 // Update updates the build on server. Returns the server's representation of the build and error if one occurs.
-func (c *builds) Update(build *buildapi.Build) (result *buildapi.Build, err error) {
-	result = &buildapi.Build{}
-	err = c.r.Put().Namespace(c.ns).Resource("builds").Name(build.Name).Body(build).Do().Into(result)
-	return
+func (c *builds) Update(build *buildapi.Build) (*buildapi.Build, error) {
```

Comment:
@mfojtik i see the use for naked returns, but in this case, we say result, then we proceed to init result to an empty struct. kinda pointless?